.github/workflows/pull-request.yml

```
@@ -187,6 +191,25 @@
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.9
+      
+      - name: Run Python script
+        if: steps.changed-files.outputs.only_changed != 'true'
```

actionlint (1.7.4)

property "changed-files" is not defined in object type {}

(expression)